README.md

```diff
@@ -43,11 +43,11 @@ Please fill out [this request form](https://docs.google.com/forms/d/e/1FAIpQLSck
 * CNCF and Bevy have organized the training session for the Community Groups organizers. It has been recorded and now [available on YouTube](https://www.youtube.com/watch?v=_rBdomoYlmc).
 * You can also access the Bevy Help Portal at <https://help.bevylabs.com/>.
 * [Bevy FAQ can be found here](https://github.com/cncf/communitygroups/blob/main/FAQ.md).
-* If you want to test video features, you can use: https://tokbox.com/developer/tools/precall/
+* If you want to test video features, you can use: [https://tokbox.com/developer/tools/precall/](https://tokbox.com/developer/tools/precall/)
```

Comment:
Wrap bare URLs in `<>`. See markdown lint rule [MD034](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md).

Feel free to accept this code suggestion per <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes>

```suggestion
* If you want to test video features, you can use: <https://tokbox.com/developer/tools/precall/>
```

@AudMonte01 we should probably add a markdown link GitHub Action to this repo so these checks are automatic 🙂